### PR TITLE
Add files upload

### PR DESCRIPTION
**Pull request by Adam2221114**

### CODE REVIEW

**Comment by Adam2221114** [comment]
.env file must add  
EMAIL_USERNAME=testemail885599@gmail.com
EMAIL_PASSWORD=nfckwebkqdwxuvus

File: server/controllers/userController.js [hunk 1]
```diff
@@ -1,107 +1,139 @@
...
+        res.json({ msg: 'Email verification is successful and the account has been created.' });
     } catch (err) {
         console.error(err.message);
         res.status(500).send('Server Error');
     }
 };
 
+
+
 exports.getUserProfile = async (req, res) => {
```
[the remark this answers is not in the document]
Comment (Adam2221114) [reply server/controllers/userController.js:90]: The characters were modified to the original "User registered successfully"

File: server/controllers/userController.js [hunk 2]
```diff
@@ -1,107 +1,139 @@
...
+        const user = await User.findById(req.user.id).select('-password');
+        res.json({ user });
+    } catch (error) {
+        console.error('Error fetching user profile:', error);
+        res.status(500).json({ msg: 'Server Error' });
+    }
 };
 
 exports.deleteAccount = async (req, res) => {
-  try {
```
[the remark this answers is not in the document]
Comment (Adam2221114) [reply server/controllers/userController.js:58]: Modify to original code

File: server/controllers/userController.js [hunk 3]
```diff
@@ -1,107 +1,139 @@
...
+    try {
+        await User.findByIdAndDelete(req.user.id);
+        res.json({ msg: 'Account deleted successfully' });
+    } catch (error) {
+        console.error('Error deleting account:', error);
+        res.status(500).json({ msg: 'Server Error' });
+    }
 };
 
 exports.updateUserProfile = async (req, res) => {
```
[the remark this answers is not in the document]
Comment (Adam2221114) [reply server/controllers/userController.js:110]: Isn't this the same thing? It doesn't seem to have changed.

File: server/controllers/userController.js [hunk 4]
```diff
@@ -1,107 +1,139 @@
...
-    await user.save();
-
-    res.json({ msg: 'User profile updated successfully' });
-  } catch (error) {
-    console.error('Error updating user profile:', error);
-    res.status(500).json({ msg: 'Server Error' });
-  }
 };
 
 exports.getUserLogs = async (req, res) => {
```
[the remark this answers is not in the document]
Comment (Adam2221114) [reply server/controllers/userController.js:130]: Are you referring to the updateUserProfile method?

File: server/controllers/userController.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (OfficialTomJ) [reply server/controllers/userController.js:110]: It's the indentation that's the issue

File: server/controllers/userController.js (hunk 4 above)
[the remark this answers is not in the document]
Comment (OfficialTomJ) [reply server/controllers/userController.js:130]: The getUserLogs method

**Pull request closed without merging**